src/cljs/rems/application.cljs

```diff
@@ -838,6 +837,9 @@
                                                        (render-events application (take 1 events))))
                               :collapse [:div
                                          [application-state-details application config]
+                                         ;; NB: possibly all handling users could use the above handler component
+                                         (when (is-handling-user? application)
+                                           [votes-summary application])
```

Comment:
As discussed today, I will make sure voting block is always shown, but otherwise handling users will generally see the slimmed state initially and only handler the full state.